notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -51,7 +51,15 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    fp16_suffix = "FP16" if not (Path(model_id.value) / "FP16_calibrated").exists() or "GPU" not in device.value else "FP16_calibrated"
</pre><p>does calibration gives any impact on cpu execution?</p><p>Maybe we can just calibrate it by default and rewrite fp16 model. it is huge model and storing 2 versions of FP16 models on disk will requires additional 5Gb disk space.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>calibration marks do not impact CPU. But calibration can be done only if GPU device available. But I agree, saving it back to the same xml would be preferable. I tried to save to the same place but execution fails (most probably because of mmap).</p><p> </p><p>Trying to turn off mmap and save to the same place.</p>

Reply:
Communicated personally. And decided that saving into a different folder and then renaming it back to 'FP16'/'INT8_compressed_weights'.